frontend/src/components/common/form/InputGroup.tsx

```diff
@@ -94,22 +94,13 @@ export const InputGroup: React.FC<InputGroupProps> = ({
         </ColPrepend>
       )}
       <ColContent>
-        {fast ? (
-          <FastInput
-            formikProps={formikProps}
```

Comment:
Seems like the only usage of formikProps have been deleted from InputGroup implementation. Maybe would be helpful to remove from interface as well? That way no need to pass ` formikProps={null as any}` ?